ext/maxminddb.c

```diff
@@ -390,14 +443,23 @@ static void handle_uint64(const MMDB_entry_data_list_s *entry_data_list,
     // We return it as a string because PHP uses signed longs
     char *int_str;
     spprintf(&int_str, 0, "%" PRIu64,
-             entry_data_list->entry_data.uint64 );
+             entry_data_list->entry_data.uint64);
     CHECK_ALLOCATED(int_str);
 
-    ZVAL_STRING(z_value, int_str, 0);
+    _ZVAL_STRING(z_value, int_str);
+    efree(int_str);
 }
 
 static zend_class_entry *lookup_class(const char *name TSRMLS_DC)
 {
+#ifdef ZEND_ENGINE_3
+    zend_string *n = zend_string_init(name, strlen(name), 0);
```

Comment:
I think you need to set `ZEND_DONT_UNLOAD_MODULES=1` as otherwise the module is unloaded and the relevant frames are not included in the trace.